packages/mds-api-server/api-server.ts

```diff
@@ -23,9 +24,11 @@ export interface ApiServerOptions {
   prometheus: PrometheusMiddlewareOptions
 }
 
-export const ApiServer = (
-  api: (server: express.Express) => express.Express,
-  options: Partial<ApiServerOptions> = {},
+export const ApiServer = <T extends {} = {}>(
+  // The linter does not realize that the type variable is used.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  api: <G = T>(server: express.Express) => express.Express,
```

Comment:
I think that this could be rewritten without having the `<T extends {} = {}>` in the `ApiServer`, and instead you could just have that exact type annotation in the `(api: ...` section without ever including `G`. I think that eslint is correct that right now, `G` _isn't_ being used, you're only using `T`

Reply:
I've tried that before, and the linter just thinks that the `T` in `api<T>` at the bottom can't be found. I think the generic type has to be inserted in `ApiServer`.